manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: NewTek.NDI5Runtime
+PackageVersion: 5.5.2.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.ndi.tv/SDK/NDI_SDK/NDI%205%20Runtime.exe
+  InstallerSha256: 8D2E8D32C62F4C34809EF77DF432B85DE35978B12358DF19889D723D0579F2BC
+  ProductCode: '{CF8E6E00-9C03-4440-81C0-21FACB921A6B}'
```

```suggestion
  ProductCode: '{71AFF296-ED43-4166-8301-4649285EE712}_is1'
```
Installer writes uninstall entries to this location in the registry

---

Indeed, I did cross-check on my machine. 
Full path if this is needed later :
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{71AFF296-ED43-4166-8301-4649285EE712}_is1